packages/ignite-dev-screens/index.js

```diff
@@ -1,27 +1,4 @@
-const copyDevScreens = async function (context) {
-  // grab some features
-  const { ignite } = context
-
-  const screens = ['APITestingScreen', 'ComponentExamplesScreen', 'DeviceInfoScreen', 'DevscreensButton', 'PluginExamplesScreen', 'PresentationScreen', 'ThemeScreen', 'ButtonBox']
-
-  const jobs = []
-
-  screens.map((screen) => {
-    jobs.push(
-      {
-        template: `${screen}.js.ejs`,
-        target: `ignite/DevScreens/${screen}.js`
-      },
-      {
-        template: `${screen}Style.js.ejs`,
-        target: `ignite/DevScreens/Styles/${screen}Style.js`
-      }
-    )
-  })
-
-  // make the templates
-  await ignite.copyBatch(context, jobs, {})
-}
+const sourceFolder = `${process.cwd()}/node_modules/ignite-dev-screens/templates`
```

Comment:
This isn't related to this PR necessarily, but I think we should replace any instances of `process.cwd()` with `SOURCE_PATH` or something, and set it further up the file. I never really know what `process.cwd()` refers to. cc @GantMan @skellock for discussion